xlsx対応

uedaueo opened the pull request:
去年1度マージして頂いているようですが，その時点ではまだmasterにマージされて居なかったようです．
ちゃんとマージして動作確認したものを作りましたので，こちらをpullして頂けますととても嬉しいです．

なお，自動生成されるファイルはバージョン管理から外してしまいました．必要でしたら再度いれておいてくださいまし．
お手数をおかけして恐縮ですが，よろしくお願いいたします．

JunjiAmanoSigrest committed:
.gitignoreへ /blanco.java/main 追記

JunjiAmanoSigrest committed:
1.8to1.7

JunjiAmanoSigrest committed:
jxl2poi

uedaueo committed:
Correct versions.

uedaueo committed:
Remove auto generated files from version control.

uedaueo committed:
Just recreated jar file.

uedaueo committed:
Adapt package specification on extends.

uedaueo committed:
Create new jar for extends package support.

igapyon reviewed (approved):
I want to receive those changes.

uedaueo committed:
Adapt annotations for Java.

uedaueo committed:
Make new jar file as blancovalueobject-2.0.2.jar

uedaueo committed:
Change annotation delimiter to double backslash.
Because comma is used as delimiter of arguments.

uedaueo committed:
Now we can specify 'import' command.
from now on, Commit generated files also.

uedaueo committed:
Merge remote-tracking branch 'upstream/master'

uedaueo committed:
Adapt full maven build.
Change meta files from xls into xlsx.
Preparing 2.0.4, targetstyle for maven and gradle

uedaueo committed:
Adapt output directory style as maven and free.

uedaueo committed:
Adapt datetime type as java.util.Date, but not support default value.
Adapt default value on php style sheet.

uedaueo committed:
Update README.md

uedaueo committed:
Adapt createToString and adjustMethodName on PHP type sheet.
